src/components/AboutUs.tsx

```diff
@@ -114,7 +110,7 @@ const AboutUs = () => {
   const { t } = useTranslation();
 
   return (
-    <Main>
+    <Flex flex="auto">
```

Comment:
I think this component has lost its flex-direction somewhere along the way...it's strange though because I can see that you've added `flex-direction: column` to Global Style

<img width="1438" alt="Screenshot 2020-11-23 at 22 05 36" src="https://user-images.githubusercontent.com/53219789/100015290-09785900-2dd8-11eb-9a5f-45da2483b42e.png">

Reply:
Right, so `flex-direction: column;` in globalStyle is bascially saying that the header is a row, and then the content is a row beneath it.

Within the route component `AboutUs.tsx`, the root `Flex` element in that component did _not_ have `flex-direction: column;`, which i've not added -- good spot!